.github/scripts/aws_db_setup.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+aws configure set aws_access_key_id $AWS_ACCESS_KEY_ID
+aws configure set aws_secret_access_key $AWS_SECRET_ACCESS_KEY
+aws configure set default.region "us-east-1"
+
+SESSION=$(AWS_PAGER="" aws sts assume-role --region us-east-1 --role-arn $ROLE_ARN --role-session-name terraform --region us-east-1 --output json)
+unset AWS_PROFILE
+
+unset AWS_ACCESS_KEY_ID
+export AWS_ACCESS_KEY_ID=$(echo $SESSION | jq -r ".Credentials.AccessKeyId")
+
+unset AWS_SECRET_ACCESS_KEY
+export AWS_SECRET_ACCESS_KEY=$(echo $SESSION | jq -r ".Credentials.SecretAccessKey")
+
+unset AWS_SESSION_TOKEN
+export AWS_SESSION_TOKEN=$(echo $SESSION | jq -r ".Credentials.SessionToken")
+
+KEY=$( aws secretsmanager get-secret-value --secret-id arn:aws:secretsmanager:us-east-1:077346344852:secret:db-ec2-key-pair-aofYm8  | jq --raw-output '.SecretString' )
+DB_PASSWORD=$( aws secretsmanager get-secret-value --secret-id arn:aws:secretsmanager:us-east-1:077346344852:secret:db-creds-WszNCl  | jq --raw-output '.SecretString' | jq -r .password )
+DB_USERNAME=$( aws secretsmanager get-secret-value --secret-id arn:aws:secretsmanager:us-east-1:077346344852:secret:db-creds-WszNCl  | jq --raw-output '.SecretString' | jq -r .username )
+DB_HOST=$( aws rds describe-db-instances --region=us-east-1 --db-instance-identifier "test-mantarray-rds-one" | jq --raw-output ".[][].Endpoint.Address" )
```

Comment:
for the secretsmanager arn, key pair arn and db name can those be an input into this script so the workflow can pass the correct one based on the terraform output or repo secret?

Reply:
Yes! Since we create those secrets manually, probably be best to pass them down as repo secrets and same with the db..